app/editor/src/components/form/select/styled/Select.tsx

```diff
@@ -1,11 +1,29 @@
 import styled from 'styled-components';
 import { Col } from 'tno-core/dist/components/flex';
 
-export const Select = styled(Col)`
+export const Select = styled<any>(Col)`
```

Comment:
Don't use `any`, it causes problems later and is hard to mainain.

Reply:
Oops thought I removed that, was playing with props at one point